lib/ansible/cli/galaxy.py

```diff
@@ -705,12 +710,13 @@ def _require_one_of_collections_requirements(self, collections, requirements_fil
             for collection_input in collections:
                 requirement = None
                 if os.path.isfile(to_bytes(collection_input, errors='surrogate_or_strict')) or \
-                        urlparse(collection_input).scheme.lower() in ['http', 'https']:
+                        urlparse(collection_input).scheme.lower() in ['http', 'https'] or \
+                        collection_input.startswith('git+'):
```

Comment:
```suggestion
                        collection_input.startswith('git+') or \:
                        collection_input.startswith('git@'):
```
Otherwise `ansible-galaxy collection install git@github.com:ansible-collections/community.azure.git` does not work:
```
ERROR! - command /usr/bin/git clone git@github.com git@github.com failed in directory /.../tmp/ansible-local-329971lp8my187/tmpi20ny9am (rc=128) - fatal: repository 'git@github.com' does not exist
```
(It does work when prepending `git+`. But just using the clone URL github spits out is nicer :) )

Reply:
Thanks, glad you spotted that